javaScript/popup.js

```diff
@@ -0,0 +1,301 @@
+  }
+
+  document.querySelector('.cards-desktop').innerHTML += `
+  <div class="card-d-1" id='work-${works[i].id}' >
+  <img class="snapshot-desktop${works[i].id} project-img" src='${works[i].image}'>
+  <div class="description">
+   <h2>${works[i].name}</h2>  
+   <div>
+    <ul class="container">
+       ${experianceList}
+    </ul>
+   </div>
+   <p>${works[i].description}</p>
+   <div>
+        <ul class="languages">
+        ${technology}
+        </ul>
+    </div> 
+    
+ <button class="see-more" type="button" data-index="${i}">See Project</button>
+ </div>
+   
+</div>`;
+
+  document.querySelector('.cards').innerHTML += `
+ <div class="card-${works[i].id} card"> 
+ <img class="snapshot-mobile${works[i].id}"  src='${works[i].image}' />
+ <div>
+   <h2>${works[i].name}</h2>
+   <div> 
+   <ul class="container">
+     ${experianceList}
+     </ul>
+   </div>
+   <p>
+     ${works[i].description}
+   </p>
+   <div>
+     <ul class="languages">
+     ${technology}
+     </ul>
+   </div>
+   <button type="button" class="see-project" data-index="${i}">See Project</button>
+ </div>
+ </div>
+ `;
+}
+
+const popModel = document.querySelector('.project-container');
+const body = document.querySelector('body');
+const displayPopupModal = (i) => {
+  let popupTech1 = '';
+  let popupTech2 = '';
+  let experianceList = '';
+
+  for (let j = 0; j < works[i].popuplanguage.length; j += 1) {
+    if (j <= 3) {
+      popupTech1 += `<li >${works[i].popuplanguage[j]}</li>`;
+    } else {
+      popupTech2 += `<li >${works[i].popuplanguage[j]}</li>`;
+    }
+  }
+
+  for (let k = 0; k < works[i].experiance.length; k += 1) {
+    experianceList
+      += k === 0
+        ? `<li><h3>${works[i].experiance[k]}</h3></li>`
+        : `<li class="counter"><img src="images/Counter.png" alt="Backend dev">${works[i].experiance[k]}</li>`;
+  }
+  body.style.overflow = 'hidden';
+  document.querySelector('.project-container').innerHTML = `
+    <div class='popup-item' id='works-${works[i].id}'>
+    <div class='popup-header'>
+    <div class="popup-header-left">
+    <h2>${works[i].name}</h2>
+    <ul class="container">
+    ${experianceList}
+    </ul>
+    </div> 
+    <i onclick="closePopupModal()" class="fa-solid fa-xmark close-project" id="close"></i>
+    </div>
+    <img src='${works[i].image}' class="hero-image">
+    <div class='description'> 
+    <div>
+    </div>
+    <div class="popup-footer">
+    <p>${works[i].popDescription}</p>
+    <div class="popup-footer-right">
+    <ul class='languages'>
+    ${popupTech1}
+   </ul>
+   <ul class='languages'>
+   ${popupTech2}
+  </ul>
+  <div class="btn-tags">
+  <button class="see-more"><a class="see"  href="${works[i].seeLive}">See Live</a><i class="fa fa-external-link" aria-hidden="true"></i>
+  </button>
+  <button  class="see-more"><a class="see" href="${works[i].seeSource}">See Source</a><i class="fa fa-github-square" aria-hidden="true"></i>
```

Comment:
the see live and see the repo buttons should be in the center in the mobile screens as in the Figma design with divider above
so please try sticking to the Figma design 

- [ ] Also add some margin between text and icon on buttons

- [ ] Figma 
![image](https://user-images.githubusercontent.com/61790587/169365195-4bba583f-d283-4d44-81d4-4e212976a7de.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/169365223-89bf2216-57e2-4ceb-a323-375bc0c594f8.png)